springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/components/DefaultComponentsScanner.java

```diff
@@ -17,28 +24,79 @@
 public class DefaultComponentsScanner implements ComponentsScanner {
 
```

Comment:
Wire the `AsyncApiDocket` here instead of in `AbstrctChannelScanner`, `AbstractMethodLevelListener` and `ClassLevelKafkaListenerScanner`.